File: python_Programmers/1단계/짝수와홀수.py
def solution(num):
    if num % 2 ==0:
        return 'Even'
    return 'Odd'

def solution2(num):
    answer = ["Even", "Odd"][num & 1]
    print(num & 1)
    return answer
num = 5
print(solution(num))
print(solution2(num))

File: java/programmers/1단계/src/짝수와홀수.java
public class 짝수와홀수 {
    public String solution(int num) {
        return (num % 2 == 0) ? "Even" : "Odd";
    }
    public static void main(String[] args) {
        짝수와홀수 sol = new 짝수와홀수();
        int num = 3;
        System.out.println(sol.solution(3));
    }
}
